src/OVAL/probes/independent/xmlfilecontent.c

```diff
@@ -128,12 +128,18 @@ static int process_file(const char *path, const char *filename, void *arg)
 		whole_path[path_len] = FILE_SEPARATOR;
 		++path_len;
 	}
+	dI("%s", path);
```

Comment:
leftover debug message. If you want it there add some actual message.

Reply:
OK